docs/usage/dex.md

```diff
@@ -0,0 +1,91 @@
+- [Overview](#overview)
+- [Dex OpenShift OAuth Connector](#dex-openshift-oauth-connector)
+    - [Role Mappings](#role-mappings)
+- [Dex GitHub Connector](#dex-github-connector)
+- [Disable DEX](#disable-dex)
+
+## Overview
+
+Dex can be used to delegate authentication to an external identity providers like GitHub, SAML and others. SSO configuration of Argo CD requires updating the Argo CD CR with [Dex connector](https://dexidp.io/docs/connectors/) settings.
```

Comment:
> to an external identity providers

_Nitpick_: This should be either "to external identity providers" or "to an external identity provider"